### PR TITLE
Refactor prompt route

### PR DESCRIPTION
**Pull request by H-b8**
- Moved `prompt` folder inside `workshop/[wid]/` folder for both `pages` and `api`.
- Refactored fetch routes for prompts and prompt throughout app.

Resolves #79

### CODE REVIEW

**Comment by MilcahHalili** [comment]
@H-b8 I keep forgetting to let you know, that I'm editing the description so it reads, `Resolves #79` because otherwise `Resolves issue #79` doesn't close the issue automatically when I close the PR!

File: pages/workshop/[wid]/prompt/new.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const NewPrompt = () => {
     }
 
     console.log(data, 'data from create prompt')
-    const res = await fetch('/api/prompt/create', {
```
Comment (MilcahHalili) [inline pages/workshop/[wid]/prompt/new.js:25]: Thanks for refactoring this!

File: pages/workshop/[wid]/prompt/new.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const NewPrompt = () => {
     }
 
     console.log(data, 'data from create prompt')
```
Comment (MilcahHalili) [inline pages/workshop/[wid]/prompt/new.js:24]: Normally I'd request to remove this console log but going to leave it for now so we can just merge!

**Review by MilcahHalili — APPROVED** [review]

**Comment by MilcahHalili** [comment]
Oops, merge conflict, @H-b8! Can you handle it, please? Or would you let me to? No worries either way! Just going to wait until the morning to if so.

File: pages/workshop/[wid]/prompt/new.js (hunk 2 above)
Comment (H-b8) [reply pages/workshop/[wid]/prompt/new.js]: @MilcahHalili Removed!

**Pull request merged**